<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="loadObjFileBtn">
       <property name="text">
        <string>打开obj文件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exportPbrtV3Btn">
       <property name="text">
        <string>生成pbrt-v3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upgradePbrtV4Btn">
       <property name="text">
        <string>升级为pbrt-v4</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="renderBtn">
       <property name="text">
        <string>渲染图片</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>loadObjFileBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>loadObjFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>47</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exportPbrtV3Btn</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>exportPbrtV3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>renderBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>render()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upgradePbrtV4Btn</sender>
   <signal>clicked()</signal>
   <receiver>MainDialog</receiver>
   <slot>upgradeToPbrtV4()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadObjFile()</slot>
  <slot>exportPbrtV3()</slot>
  <slot>render()</slot>
  <slot>upgradeToPbrtV4()</slot>
 </slots>
</ui>
